roles/set_openstack_containers/defaults/main.yml

```diff
@@ -25,6 +25,7 @@ cifmw_set_openstack_containers_prefix: openstack
 cifmw_set_openstack_containers_operator_name: openstack
 cifmw_set_openstack_containers_tag_from_md5: false
 cifmw_set_openstack_containers_dlrn_md5_path: "/etc/yum.repos.d/delorean.repo.md5"
+cifmw_set_openstack_containers_openstack_version_change: false
```

Comment:
Shouldn't this make its way to the readme ?

Reply:
in readme together with readme